seshdash/models.py

```diff
@@ -136,11 +136,12 @@ def save(self, *args, **kwargs):
         else:
             super(Sesh_Site, self).save(*args, **kwargs)
 
-    def delete(self):
+    def delete(self, *args, **kwargs):
         # Delete the site and its associated status card
         status_card = self.status_card
+        print "Deleting associated status card"
```

Comment:
can we stop using prints and always use the logger? This goes for all the other changes. Not sure if the ones in the view.py are yours